ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -138,6 +104,15 @@ SettingsPageLayout {
                     stackManager.stackView.replace(welcomeView, mintedTokensView, StackView.Immediate)
             }
         }
+
+        onInitialItemChanged: updateInitialStackView()
+    }
+
+    QtObject {
+        id: temp_
+
+        property CollectibleObject collectible: CollectibleObject{}
```

Comment:
whenever it's not intended to have collectible/asset reassigned, imo it should be marked as `readonly` for clarity.